### PR TITLE
Advertise SINGLE_NODE_MULTI_WRITER capability

### PR DESCRIPTION
**Pull request by chrishenzie**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Updates the CSI spec version to v1.5.0, which includes support for `SINGLE_NODE_SINGLE_WRITER` and `SINGLE_NODE_MULTI_WRITER` access modes.

Also advertises the `SINGLE_NODE_MULTI_WRITER` capability so CSI sidecars can supply the appropriate access mode.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @pohly 
/assign @xing-yang

### CODE REVIEW

File: pkg/hostpath/controllerserver.go [hunk 1]
```diff
@@ -796,6 +796,7 @@ func (hp *hostPath) getControllerServiceCapabilities() []*csi.ControllerServiceC
 			csi.ControllerServiceCapability_RPC_LIST_VOLUMES,
 			csi.ControllerServiceCapability_RPC_CLONE_VOLUME,
 			csi.ControllerServiceCapability_RPC_VOLUME_CONDITION,
+			csi.ControllerServiceCapability_RPC_SINGLE_NODE_MULTI_WRITER,
```
Comment (pohly) [inline pkg/hostpath/controllerserver.go:799]: Would it make sense to make this configurable, to enable testing with and without this capability?

Just thinking aloud here, I don't know whether there will be any tests that depend on this.

**Review by pohly — COMMENTED** [review]
Tests are failing. I haven't checked why.

**Comment by pohly** [comment]
> Tests are failing. I haven't checked why.

Well, that was easy: csi-sanity must be updated first to accept the new capability.

**Comment by chrishenzie** [comment]
Rebased on master now that the CSI sanity test suite version has been bumped. Hoping this works!

**Review by pohly — APPROVED** [review]
/lgtm
/approve

**Pull request merged**